### ISSUE TITLE
RDP clipboard not working with Server 2025

### ISSUE THREAD

**Issue by slowreload** [mRemoteNG/mRemoteNG#2653]
When connected to RDP to a server 2025 server RDP clipboard doesnt work, Not able to copy in or out. Works with all other OS Win10/11 Server16-22. Copy paste also works locally on the remote machine from within the rdp session and when using mstsc.exe just not with mremote.

## Expected Behavior
Should be able to copy in and out normally with RDP clipboard.

## Current Behavior
Clipboard is no longer shared from host or remote machine when using mRemote. Works with MSTSC.exe and can copy normally.

## Steps to Reproduce (for bugs)
1. Log in to server 2025 machine
2. Copy text/file on remote machine
3. Paste on local machine 
4.Copy text/file on Local Machine
5.Paste on Remote Machine

## Context
When following documentation or scripts its is very important to be able to copy and paste text such as commands or keys to prevent fat fingers/typos and save a bunch of time. Copy paste has been something that is a critical fucntion that is used all the time 

## Your Environment
* Version used: 10.77.3
* Operating System and version (e.g. Windows 10 1709 x64): Win10 22H2 X64 Build 19045.5011

**Comment by simonai1254** [comment]
Did you verify, that in the connection settings RDP is enabled (see https://github.com/mRemoteNG/mRemoteNG/issues/1562#issuecomment-528502502)?

**Comment by slowreload** [comment]
Yes I did. The server was originally 2016 then was in place upgraded to 2025 and it worked when it was 2016. I also tried on a fresh build and it was the same effect. I tried rolling back to the  last stable version 1.76.20 and used the same connection file with zero changes and it works as intended it is a issue with version 1.77.3

**Comment by Kvarkas** [comment]
will check

**Comment by tecxx** [comment]
i just tried this from a win10 machine to a server 2025 machine and had no issues, but i'm using the latest source build, based on .net 8, and RDC12 commented out. not sure if this info helps.

**Comment by JaivigneshJv** [comment]
https://woshub.com/cant-copy-paste-via-remote-desktop-rdp-clipboard/

closed and reopened remote clipboard monitor

**Comment by russia** [comment]
> https://woshub.com/cant-copy-paste-via-remote-desktop-rdp-clipboard/
> 
> closed and reopened remote clipboard monitor

That fixed the issue on my end, thank you.

**Comment by robertpopa22** [comment]
Triage consolidation: this thread already contains signals that the problem may not reproduce on current code paths. 
 
- contributor check on latest source build reported no repro 
- multiple users reported clipboard reset workaround (`rdpclip.exe` restart) restores expected behavior 
 
Recommendation: keep `Need 2 check` only if someone can reproduce on latest dev/fork builds with a deterministic step list (host OS/client OS/RDP options, especially clipboard redirection). Otherwise, close as not reproducible/stale.
